README.md

```diff
@@ -125,8 +125,12 @@ When configuring CollectD, the target URL path will be `/post-collectd`.
 
 #### signalfx
 
-You can pretend to be a SignalFx endpoint with the `signalfx` type.  For this,
-you will need to specify which port to bind to.  An example config:
+You can pretend to be a SignalFx endpoint with the `signalfx` type. A SignalFx
```

Comment:
I wouldn't use the term "pretend to be..." i would use "You can expose a SignalFx endpoint..."